tick/plot/plot_stem.py

```diff
@@ -31,18 +38,18 @@ def stem(y: np.ndarray, show=True, title=None, x_range=None, y_range=None):
                     x_range=x_range, y_range=y_range)
     fig.scatter(x, y, size=4, fill_alpha=0.5)
     fig.segment(x, np.zeros(dim), x, y)
-    fig.title_text_font_size = "12pt"
+    fig.title.text_font_size = "12pt"
     if title is not None:
-        fig.title = title
+        fig.title.text = title
     if show:
         bk.show(fig)
-        return None
+        return fig
```

Comment:
You say in the docstring you're doing so but in the code you are returning `fig`